code/modules/spacepods/spacepod.dm

```diff
@@ -589,11 +589,21 @@ obj/spacepod/proc/add_equipment(mob/user, var/obj/item/device/spacepod_equipment
 			desc = "A rough looking space pod meant for industrial work"
 	update_icons()
 
+/obj/spacepod/proc/can_use_verb()
```

Comment:
Declaring a proc is just declaring a public function. In other words, it is globally available.